NitroxClient/GameLogic/DevConsoleSpawnEvent.cs

```diff
@@ -0,0 +1,58 @@
+﻿using System;
+using System.Collections.Generic;
+using NitroxClient.Communication.Abstract;
+using NitroxClient.GameLogic.Helper;
+using NitroxModel.DataStructures;
+using NitroxModel.DataStructures.GameLogic;
+using NitroxModel.Logger;
+using NitroxModel.Packets;
+using NitroxModel_Subnautica.Helper;
+using UnityEngine;
+
+namespace NitroxClient.GameLogic
+{
+    public class DevConsoleSpawnEvent
+    {
+        private readonly IPacketSender packetSender;
+        private readonly Vehicles vehicles;
+
+        public DevConsoleSpawnEvent(IPacketSender packetSender, Vehicles vehicles)
+        {
+            this.packetSender = packetSender;
+            this.vehicles = vehicles;
+        }
+
+        /// <summary>
+        /// List of things that can be spawned : https://subnauticacommands.com/items
+        /// Vehicle that can be spawn : Seamoth, Exosuit
+        /// </summary>
+        public void Spawn(GameObject gameObject)
+        {
+            try
+            {
+                List<InteractiveChildObjectIdentifier> childIdentifiers = VehicleChildObjectIdentifierHelper.ExtractInteractiveChildren(gameObject);
+                TechType techtype = CraftData.GetTechType(gameObject);
+
+                ConstructorBeginCrafting constructorBeginCrafting = VehicleHelper.BuildFrom(gameObject, techtype, childIdentifiers, new NitroxId(), 0f);
+                VehicleModel vehicleModel = VehicleModelFactory.BuildFrom(constructorBeginCrafting);
```

Comment:
Is it possible for us to refactor so that we don't need to build a ConstructorBeginCrafting first?  Potentially use the logic to build a VehicleModel which can be used to construct ConstructorBeginCrafting above.